docs/getting-started/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Getting Started with Octopus Deploy
+description: This section provides instructions to help you with your first deployment.
+position: 1
+---
+
+This Getting Started series walks you through a simple deployment with Octopus Deploy and explains the Octopus Deploy fundamentals so that you will be equipped to take on more complicated deployments by the end of the series.
+
+## 1. Hosting the Octopus Deploy Server
+
+The first thing to decide is where to host your [Octopus Deploy server](/docs/installation/index.md). You can install your own [self-hosted](/docs/installation/index.md) instance or use [Octopus Cloud](/docs/octopus-cloud/index.md) which is hosted and maintained by Octopus Deploy.
+
+### **Installing the Self-hosted Octopus Deploy Server**
+1. Download the installer from [{{Resources,Downloads}}](https://octopus.com/downloads)
+1. Run the installer on your machine and follow the [installation instructions](docs/installation/index.md#install-octopus).
+1. Once installed, click **Get started...** and follow the installer wizard to set up your instance.
+1. Click **Install**.
+
+### **Create a Cloud Instance**
+1. Create an account by clicking **Start free trial** at [octopus.com/trial](https://octopus.com/trial)
+1. From the instances screen, click **Create cloud instance** and follow the [set up instructions](docs/octopus-cloud/index.md#getting-started-with-octopus-cloud).
+1. Confirm the details you've provided, agree to the terms and click **Looks good. Deploy my Octopus!**.
+
+## 2. Configure your Deployment Environments
+
+Octopus needs to connect to the [Infrastructure](/docs/infrastructure/index.md) you'll be using to deploy your software. You can deploy to Windows servers, Linux servers, Microsoft Azure, an Offline Package Drop, Cloud Regions, or Kubernetes. These are known as your deployment targets, and they are organized into environments so you can promote your software through your deployment pipeline, for instance, from Development to Testing and finally into Production.
```

Comment:
Change "Testing" to "Test" in "Development to Testing and finally into Production"